lock/src/main/java/com/auth0/android/lock/views/ClassicPanelHolder.java

```diff
@@ -357,11 +388,18 @@ public void onKeyboardStateChanged(boolean isOpen) {
         if (customFieldsForm != null) {
             customFieldsForm.onKeyboardStateChanged(isOpen);
         }
+        if (mfaCodeFormView != null) {
```

Comment:
Do you suggest using the NullObject pattern? i.e. `Form` interface with `onKeyboardStateChanged`, `ValidForm` and `NullForm` implements it. At the beginning, all my forms are `NullForm`'s. But I always interact with a `Form` object.